src/views/minor/Footer.vue

```diff
@@ -1,20 +1,38 @@
 <template>
     <footer class="page-footer font-small">
-        <div class="footer-copyright text-center py-3" style="color: white">
+        <div class="text-center py-3" style="color: white">
             {{ copyright }}
+            <p>
+                Licensed under the
+                <a
+                    class="footer-link"
+                    href="https://opensource.org/licenses/MIT">
+                    MIT License</a
+                >.
+            </p>
+            <p>
+                By using this website, you agree any images you create are
+                released into the public domain according to
```

Comment:
"By using this website, you agree that any images or media you create using the tools on this website are released by you into the public domain according to CC0."  Good catch about the word "license".  I still wanted to update the rest of the wording a bit, to include other media (we may have sound for example, or video), and to indicate that YOU are doing the releasing.

Reply:
Whoops, I missed the bit about media. Adding it now...

Reply:
Should be fixed in https://github.com/numberscope/frontscope/pull/119/commits/58cc120f6d5728f466183fd5ac5caff83116d4d2.